package.json

```diff
@@ -2,14 +2,18 @@
   "name": "blog",
   "private": true,
   "dependencies": {
+    "@babel/core": "^7.0.0",
     "@rails/actioncable": "^6.0.0",
     "@rails/activestorage": "^6.0.0",
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "5.2.1",
+    "babel-loader": "^8.2.2",
     "turbolinks": "^5.2.0"
   },
   "version": "0.1.0",
   "devDependencies": {
+    "webpack": "4.44.1",
```

Comment:
Why do you need `webpack` and `webpack-cli` as dev dependencies? You already have `@rails/webpacker` as a dependency.

Reply:
I removed that `devDependencies` and i reinstalled `webpacker` and upgrade with yarn. Now is not necessary to use `YARN_PRODUCTION` environment variable.

Source: [webpacker instalation](https://github.com/rails/webpacker#installation)